linode/manager
# change: [M3-8552] - eslint - mark formik as deprecated

## Description 📝
See https://github.com/linode/eslint-plugin-cloud-manager/pull/6 for details about the new plugin

## Changes  🔄
- bump `@linode/eslint-plugin-cloud-manager` to `^0.0.4`
- Update documentation

## Preview 📷

![Screenshot 2024-09-16 at 11 00 10](https://github.com/user-attachments/assets/e759678b-587c-4d3d-9317-83a06a7158ec)

![Screenshot 2024-09-16 at 11 00 01](https://github.com/user-attachments/assets/8412b7a5-a1f6-416f-ae72-9e8fb171771d)

## How to test 🧪

### Verification steps
- Pull code locally, restart eslint in your IDE (or restart IDE) and confirm formik is now shown as deprecated
- run `yarn docs` and verify the new "Composition" section of our developer documentation

## As an Author I have considered 🤔

*Check all that apply*
